pyof/v0x01/common/flow_match.py

```diff
@@ -119,3 +120,27 @@ def __init__(self, wildcards=None, in_port=None, dl_src=None, dl_dst=None,
         self.nw_dst = nw_dst
         self.tp_src = tp_src
         self.tp_dst = tp_dst
+
+        if self.wildcards is None:
+            self.fill_wildcards()
+
+    def fill_wildcards(self):
+        """Fill wildcards considering the attributes of the instance."""
+        self.wildcards = 0
+
+        simple_fields = ['in_port', 'dl_src', 'dl_dst', 'dl_vlan',
+                         'dl_vlan_pcp', 'dl_type', 'nw_tos',
+                         'nw_proto', 'tp_src', 'tp_dst']
+
+        for field in simple_fields:
```

Comment:
@macartur , the ``GenericStruct`` class has some helper methods like ``get_attributes``, so you don't need to use this kind of hardcoded list.

It would be better to use the built-in method from our class and, if you have special cases for some attributes, then just do an if-else statement for those cases - such as ``wildcards`` attribute, ``nw_src``and so on.